File: helpup-ng-app/src/app/models/Conversation.ts
import {Chat} from './Chat';

export class Conversation {
  chat: Chat[];
  to: string;
  from: string;
  icon: any;


  constructor(to, from, icon) {
    this.chat = [];
    this.to = to;
    this.from = from;
    this.icon = icon;
  }
}

File: helpup-api/src/main/java/com/malinabenegui/help/models/Conversation.java
package com.malinabenegui.help.models;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.Vector;

@Getter
@Setter
@NoArgsConstructor
public class Conversation {
    private Vector<Chat> chat;
    private String to;
    private String from;
    private byte[] icon;

    public Conversation(String to, String from, byte[] icon) {
        chat = new Vector<>();
        this.to = to;
        this.from = from;
        this.icon = icon;
    }
}
